src/backend/distributed/metadata/metadata_utility.c

```diff
@@ -2162,3 +2162,14 @@ TableOwner(Oid relationId)
 
 	return GetUserNameFromId(userId, false);
 }
+
+
+/*
+ * IsForeignTable takes a relation id and returns true if it's a foreign table.
+ * Returns false otherwise.
+ */
+bool
+IsForeignTable(Oid relationId)
```

Comment:
One another question I have is regarding this: https://www.postgresql.org/docs/14/sql-importforeignschema.html

How would `IMPORT FOREIGN SCHEMA` work with Citus when `SET citus.use_citus_managed_tables TO ON;`?

Reply:
ping on this, it seems like a common pattern